robottelo/test.py

```diff
@@ -279,6 +280,14 @@ def setUp(self):  # noqa
             self.addCleanup(self.browser.quit)
         self.browser.maximize_window()
         self.browser.get(settings.server.get_url())
+        # Workaround 'Certificate Error' screen on Microsoft Edge
+        if (self.driver_name == 'edge' and
+                'Certificate Error' in self.browser.title or
+                'Login' not in self.browser.title):
```

Comment:
Seems like it's the only working way to bypass 'Certificate Error' screen for now. 

`'Login' not in self.browser.title` part was added as that 'Certificate Error' string may be localized (d'oh), for me it shows 'Помилка сертифікату', so in case system language is not English - better to ensure we aren't on login screen rather than search for specific certificate error phrase.